index.html

```diff
@@ -0,0 +1,180 @@
+                    </div>
+                    <ul class="nav_wrapper">
+                        <li class="nav_items" onclick="closeMobileMenu()" >
+                            <a href="./about.html" class="linkA">About</a> 
+                        </li>
+                        <li class="nav_items" onclick="closeMobileMenu()">
+                            <a href="#courses" class="link">Program</a>
+                        </li>
+                        <li class="nav_items" onclick="closeMobileMenu()" >
+                            <a href="#courses" class="link">Courses</a>
+                        </li>
+                        <li class="nav_items" onclick="closeMobileMenu()">
+                            <a href="#lecturers" class="link">Lecturers</a>
+                        </li>
+                        <li class="nav_items" onclick="closeMobileMenu()" >
+                            <a href="#partners" class="link">Partners</a>
+                        </li>
+                        <li class="nav_items" onclick="closeMobileMenu()">
+                            <a href="#" class="linkB">OW Inno.</a>
+                        </li>
+                    </ul>
+                </nav>
+                <div class="nav_wrapper">
+                    <nav class="desktop_nav">
+                        <div class="nav-logo">
+                            <a href="./index.html"><img src="images/logo-removebg-preview.png" class="nav-logo" alt="logo of Orbit Web Innovations"></a>
+                        </div>
+                        <ul class="nav_wrapper_desk">
+                            <li class="nav_items_desk">
+                                <a href="./about.html" class="link_deskA">About</a> 
+                            </li>
+                            <li class="nav_items_desk">
+                                <a href="#courses" class="link_desk">Program</a>
+                            </li>
+                            <li class="nav_items_desk">
+                                <a href="#courses" class="link_desk">Courses</a>
+                            </li>
+                            <li class="nav_items_desk">
+                                <a href="#lecturers" class="link_desk">Lecturers</a>
+                            </li>
+                            <li class="nav_items_desk">
+                                <a href="#partners" class="link_desk">Partners</a>
+                            </li>
+                            <li class="nav_items_desk">
+                                <a href="#" class="link_deskB">OW Inno.</a>
+                            </li>
+                        </ul>
+                    </nav>
+                </div>
+        </div>
+    </header>
+    <section class="hero">
+                <div class="hero_box">
+                    <h1 class="hero_heading">"Hello! Sharing World"</h1>
+                    <h2 class="hero_title">ORBIT WEB INNOVATIONS</h2>
+                </div>
```

Comment:
- [ ] I know we don't have a Figma template to work with on this project. But we should try to make the project a visual copy of the original design. Please adjust the `font-size` and `font-weight` in this section to match the template. 🤓️

| Your design                                                             | Template                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/217484875-d4315cfd-cead-41b8-84a3-27f8884f5ff8.png" width="400">|<img src="https://user-images.githubusercontent.com/17243325/217484855-6ebaa776-738e-487e-8913-df4a2be51953.png" width="400">|